tsl/test/isolation/specs/CMakeLists.txt

```diff
@@ -3,9 +3,18 @@ set(TEST_TEMPLATES_MODULE reorder_deadlock.spec.in
                           reorder_vs_insert_other_chunk.spec.in)
 
 set(TEST_TEMPLATES_MODULE_DEBUG
-    deadlock_recompress_chunk.spec.in reorder_vs_insert.spec.in
-    reorder_vs_select.spec.in remote_create_chunk.spec.in
-    dist_restore_point.spec.in continuous_aggs_drop_chunks.spec.in)
+    reorder_vs_insert.spec.in reorder_vs_select.spec.in
+    remote_create_chunk.spec.in dist_restore_point.spec.in
+    continuous_aggs_drop_chunks.spec.in)
+
+# These tests are using markers for the isolation tests (to avoid race
+# conditions causing differing output), which were added after 12.7, 13.3, and
+# 14.0.
+if(PG_VERSION VERSION_GREATER "12.7"
+   OR PG_VERSION VERSION_GREATER "13.3"
+   OR PG_VERSION VERSION_GREATER "14.0")
```

Comment:
shouldn't this be `"12.6"` and `"13.2"`? I don't know if `VERSION_GREATER_EQUAL` is available, but if it is not then we have a problem with `"14.0"`.

Reply:
For 13.3, we have
```
mats@fury:~/repos/postgres$ git describe --tags
REL_13_3-73-ge2cde85ef2
mats@fury:~/repos/postgres$ git log --oneline HEAD^..
e2cde85ef2 (HEAD) Use annotations to reduce instability of isolation-test results.
```

You're correct about the 14.0 (it was added in a beta), so fixed that.